src/app/zap-templates/templates/chip/helper.js

```diff
@@ -445,6 +446,127 @@ async function chip_endpoint_clusters(options)
   return templateUtil.collectBlocks(clusters, options, this);
 }
 
+/**
+ * If helper that checks if a type is a bitmap which is non-atomic.
+ * Non-atomic bitmaps are bitmaps which are not bitmap8/16/32(generally defined
+ * in types.xml)
+ * example:
+ * {{#if_is_weakly_typed_bitmap type}}
+ * type is non atomic bitmap
+ * {{else}}
+ * type is not a non-atomic bitmap
+ * {{/if_is_weakly_typed_bitmap}}
+ *
+ * @param {*} type
+ * @returns Promise of content.
+ */
+async function if_is_weakly_typed_bitmap(type, options)
+{
+  let packageId = await templateUtil.ensureZclPackageId(this)
+  let bitmap
+  if (type && typeof type === 'string')
+  {
+    bitmap = await queryZcl.selectBitmapByName(this.global.db, packageId, type)
+  }
+  else
+  {
+    bitmap = await queryZcl.selectBitmapById(this.global.db, type)
+  }
+
+  if (bitmap) {
+    let a = await queryZcl.selectAtomicType(this.global.db, packageId, bitmap.name)
+    if (a)
+    {
+      return options.inverse(this)
+    }
+    else
+    {
+      return options.fn(this)
+    }
+  }
+  return options.inverse(this)
+}
+
+/**
+ * An helper function to a handlebar-helper that checks if a type is an enum
```

Comment:
Again, this documentation does not match what the function's "external API" is in terms of why one would use this function.